test/index.js

```diff
@@ -1,34 +1,38 @@
 "use strict"
-var test = require("tap").test
-var requireInject = require("require-inject")
+const tap = require('tap');
+const requireInject = require('require-inject');
 
-test("Windows", function (t) {
-  t.plan(1)
-  var hasUnicode = requireInject("../index.js", {
+tap.test("Windows", function (t) {
+  const hasUnicode = requireInject("../index.js", {
     os: { type: function () { return "Windows_NT" } }
-  })
-  t.is(hasUnicode(), false, "Windows is assumed NOT to be unicode aware")
-})
-test("Unix Env", function (t) {
-  var hasUnicode = requireInject("../index.js", {
+  });
+  t.equal(hasUnicode(), false, "Windows is assumed NOT to be unicode aware");
+  t.end();
+});
+
+tap.test("Unix Env", function (t) {
+  const hasUnicode = requireInject("../index.js", {
     os: { type: function () { return "Linux" } },
-    child_process: { exec: function (cmd,cb) { cb(new Error("not available")) } }
```

Comment:
yeah, I have no idea at this point what the child_process shim is doing here either. I kinda think it dates to a prior version that had fallbacks to like, windows commands, but I don't know if that ever actually saw the light of day. Regardless, it can go.